### PR TITLE
ROC-7985: Add bulk print response to case details prod

### PR DESCRIPTION
**Pull request by dharmendrak**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-7985

### Change description ###

ROC-7985: Add bulk print response to case details prod

**Does this PR introduce a breaking change?**

```
- [ ] Yes
- [ ] No
```

### CODE REVIEW

File: definition/data/sheets/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -290,5 +290,10 @@
...
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ManageDocuments", "UserRole":"caseworker-cmc-legaladvisor", "CRUD": "R"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ManageDocuments", "UserRole":"caseworker-cmc-solicitor", "CRUD": "R"},
-  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ChangeContactDetails", "UserRole": "caseworker-cmc", "CRUD": "R"}
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ChangeContactDetails", "UserRole": "caseworker-cmc", "CRUD": "R"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "UpdateBulkPrintDetails", "UserRole": "citizen", "CRUD": "CRU"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "UpdateBulkPrintDetails", "UserRole": "caseworker-cmc-anonymouscitizen", "CRUD": "CR"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "UpdateBulkPrintDetails", "UserRole": "caseworker-cmc-judge", "CRUD": "CU"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "UpdateBulkPrintDetails", "UserRole": "caseworker-cmc-legaladvisor", "CRUD": "CU"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "UpdateBulkPrintDetails", "UserRole": "caseworker-cmc-solicitor", "CRUD": "CRU"}
```
Comment (daveoh) [inline definition/data/sheets/AuthorisationCaseEvent.json:298]: Do we want to have Create auth on this prod PR?

File: definition/data/sheets/CaseEvent.json [hunk 2]
```diff
@@ -78,5 +78,6 @@
...
-  {"CaseTypeID": "MoneyClaimCase", "ID": "ChangeContactDetails", "Name": "Change contact details", "Description": "Change the contact details for any party.", "DisplayOrder": 1,"PreConditionState(s)": "*","PostConditionState": "*", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutAboutToSubmitEvent": 0, "EndButtonLabel": "Submit Changes" }
+  {"CaseTypeID": "MoneyClaimCase", "ID": "ChangeContactDetails", "Name": "Change contact details", "Description": "Change the contact details for any party.", "DisplayOrder": 1,"PreConditionState(s)": "*","PostConditionState": "*", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutAboutToSubmitEvent": 0, "EndButtonLabel": "Submit Changes" },
+  {"CaseTypeID": "MoneyClaimCase", "ID": "UpdateBulkPrintDetails", "Name": "Bulk print Request details", "Description": "add Bulk print details for the request.", "DisplayOrder": 1,"PreConditionState(s)": "*","PostConditionState": "*", "SecurityClassification": "Public"}
```
Comment (daveoh) [inline definition/data/sheets/CaseEvent.json:82]: Is the capitalisation on the name and description correct here, i.e 
```suggestion
  {"CaseTypeID": "MoneyClaimCase", "ID": "UpdateBulkPrintDetails", "Name": "Bulk print request details", "Description": "Add Bulk print details for the request.", "DisplayOrder": 1,"PreConditionState(s)": "*","PostConditionState": "*", "SecurityClassification": "Public"}
```

File: definition/data/sheets/AuthorisationCaseEvent.json (hunk 1 above)
Comment (dharmendrak) [reply definition/data/sheets/AuthorisationCaseEvent.json:298]: yes, it's not CTSC feature, but a system feature to capture bulk print requests

File: definition/data/sheets/CaseEvent.json (hunk 2 above)
Comment (dharmendrak) [reply definition/data/sheets/CaseEvent.json]: done

**Comment by daveoh** [comment]
The new event that is in the demo PR should be in the prod PR but with no create authorisation

**Comment by akritis22** [comment]
Please resolve conflicts

File: definition/data/sheets/CaseEvent.json [hunk 3]
```diff
@@ -39,7 +39,7 @@
...
   {"CaseTypeID": "MoneyClaimCase", "ID": "SettlementAgreementUpload", "Name": "SA uploaded", "Description": "SA uploaded", "DisplayOrder": 1, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "SupportUpdate", "Name": "Support update", "Description": "Support update", "DisplayOrder": 1, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public"},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "attachScannedDocs", "Name": "Attached via bulk scan", "Description": "Attached via bulk scan", "DisplayOrder": 1, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "attachScannedDocs", "Name": "Attached via bulk scan", "Description": "Attached via bulk scan", "DisplayOrder": 1, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit"},
```
Comment (akritis22) [inline definition/data/sheets/CaseEvent.json:42]: Please add "RetriesTimeoutAboutToStartEvent": "0"

**Pull request merged**